controllers/dashboard_controller.go

```diff
@@ -736,3 +743,26 @@ func (r *GrafanaDashboardReconciler) GetMatchingDashboardInstances(ctx context.C
 
 	return instances, err
 }
+
+func (r *GrafanaDashboardReconciler) UpdateHomeDashboard(ctx context.Context, grafana v1beta1.Grafana, uid string, dashboard *v1beta1.GrafanaDashboard) error {
+	grafanaClient, err := client2.NewGrafanaClient(ctx, r.Client, &grafana)
+	if err != nil {
+		return err
+	}
+
+	p, err := grafanaClient.OrgPreferences()
```

Comment:
There's a small bug in the grafana-api-go-client that makes us unable to use the PATCH endpoint, having to GET first and then PUT. I solved it in this [PR](https://github.com/grafana/grafana-api-golang-client/pull/170), so I'm waiting for a new release to change this.